scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -150,7 +154,11 @@
                                         variant_id=variant._id) }}">
                       {% set display_genes = [] %}
                       {% for gene in variant.genes %}
-                        {{ "" if display_genes.append(gene.hgnc_symbol + ' ' + gene.hgvs_identifier) }}
+                        {% if 'hgvs_identifier' in gene %}
```

Comment:
These two (this one and the `causatives_list` above) are almost identical. Could we break out these lines into its own macro?

Reply:
mmm I don't agree so much with having this code in a separate macro here. I agree with you that it's almost the same code as in the causative list, but it's a just a few lines. 
Macros are very good when it comes to perform elaborate and repetitive tasks, but the disadvantage is that from the main HTML page you don't see what happens in them, and you kind of lose the train of thoughts (The if/else conditions) and you need to look for the code in the utils. I don't think it's worth to move this code there.

Reply:
It's also good not to fragment too much the page into macros with very little code, because it will become more and more complicated to handle, this is what I mean.